tests/test_current_mission.py

```diff
@@ -14,26 +14,33 @@
 from zeusops_bot.reforger_config_gen import ReforgerConfigGenerator, as_config_file
 
 
-def test_current_mission(mission_dir: Path, config_gen: ReforgerConfigGenerator):
+def test_current_mission(mission_dir: Path, base_config: Path):
     """Scenario: Show currently active mission"""
     # Given file "mission1.json" exists in the mission directory
     mission_name = "mission1"
     as_config_file(mission_dir, mission_name).touch()
 
     # And "mission1.json" is set as the active mission
+    config_gen = ReforgerConfigGenerator(
+        base_config_file=base_config, target_folder=mission_dir
+    )
     config_gen.zeus_set_mission(mission_name)
     # When Zeus calls "/current-mission"
     result: str = config_gen.current_mission()
     # Then "mission1" is displayed
     assert result == mission_name, "Should match previously set mission name"
 
 
-def test_broken_link(mission_dir: Path, config_gen: ReforgerConfigGenerator):
+def test_broken_link(mission_dir: Path, base_config: Path):
     """Test that a broken link as active mission raises an error"""
```

Comment:
Should likely be able to explain that as a Gherkin Scenario though?
Like it's normal to describe it as a behaviour in Gherkin, just a not happy path one

Reply:
Done